app/src/main/java/com/github/sdp/tarjetakuna/ui/webapi/WebApi.kt

```diff
@@ -96,7 +96,7 @@ open class WebApi {
      */
     fun getCardsBySet(set: String): CompletableFuture<MagicApiCards> {
         val promise = CompletableFuture<MagicApiCards>()
-        ApiCall(getMagicApi().getCardsBySet(set), promise).enqueue()
+        ApiCall(getMagicApi().getCardsBySearch("s:$set", "set"), promise).enqueue()

```

Comment:
calls like this with string parameter should be defined in the scryfall api file i think

Reply:
True. Unfortunately, I did not have time to make these changes, and I preferred to opt for a more global version. I can change that in a new PR.

Reply:
then put a todo for next work

Reply:
I looked at the code one more time, and I think it's in the right place to end. WebApi includes a list of queries, including one on name/sets search that calls /search.